### ISSUE TITLE
styles in new window blocked by strict CSP

### ISSUE THREAD

**Issue by avispeng** [rmariuzzo/react-new-window#106]
Hi,

Thanks for making this nice and easy-to-use library. But I am having issues when applying strict CSP rules to our website. The styles in the new window are blocked and not applied. We are using the nonce method to manage the style-src in CSP. The nonce is added to the style tags when doing server side rendering. I googled and couldn't find a way to handle CSP in the case that new window is opened dynamically. I am hoping I can get some ideas here. Thank you.

**Comment by rmariuzzo** [comment]
Hey there! I'm sorry for taking a year to reply! I'm not familiar with that scenario, I will need time to investigate it. Putting this issue on hold.
